addons/binding/org.openhab.binding.vitotronic/ESH-INF/thing/thing-types.xml

```diff
@@ -15,12 +15,41 @@
 
         <channels>
             <channel id="systemtime" typeId="systemtime"/>
-            <channel id="outside_temp" typeId="outside_temp"/>
-            <channel id="boiler_temp" typeId="boiler_temp"/>   
-            <channel id="flowuprating" typeId="flowuprating"/>   
+            <channel id="outside_temp" typeId="temperature_ro">
+		        <label>Outside Temperature</label>
+		        <description>Outside temperature sensor</description>
+            </channel>
+            <channel id="boiler_temp" typeId="temperature_ro_adv">
+		        <label>Boiler Temperature</label>
+		        <description>Temperature sensor of boiler (fireplace)</description>
+            </channel>   
+            <channel id="malfunction" typeId="bool_ro">   
+		        <label>Heating Malfunction</label>
+		        <description>General Malfunction state of the heating</description>
+		    </channel>

```

Comment:
Not yet my review, but could you please re-apply the code formatter on all your xml files?